dartsim/src/WorldFeatures.cc

```diff
@@ -196,6 +196,78 @@ const std::string &WorldFeatures::GetWorldSolver(const Identity &_id) const
   return solver->getBoxedLcpSolver()->getType();
 }
 
+/////////////////////////////////////////////////
+void WorldFeatures::SetWorldSolverIterations(const Identity &_id,
+    std::size_t _iterations)
```

Comment:
include `<cstddef>`

Reply:
added in header file. [adc1b3d](https://github.com/gazebosim/gz-physics/pull/609/commits/adc1b3dac83b67461cf0a67c8566540956782307)